### DISCUSSION TITLE
how to integrate into a website

### DISCUSSION THREAD

**Discussion by bharath-020** [sharkiller/Reproductor-MPD-M3U8#29] [Q&A] [answered]
after spending lot of hours in search of m3u8 players i found this one. its one the best i ever seen . none i have play m3u8 videos in the desktop browser. so can anyone help or tell me how to integrate this m3u8 player to my website . please

**Comment by angelmohi** [comment]
Hi friend, have you found a solution?

**Reply by sharkiller** [reply, marked as answer]
> Hi friend, have you found a solution?

@bharath-020 @angelmohi 

sorry i didnt saw this section on github not even got an email.

you can find how to integrate in your website in the wiki.

https://github.com/sharkiller/Reproductor-MPD-M3U8/wiki/Embed-player
